tests/test_nrnhines.py

```diff
@@ -54,6 +54,14 @@ def test_point_to_section_end():
                  3)
 
 
+def test_NeuroM_section_to_NRN_section():
+    mapping = tested.NeuroM_section_to_NRN_section(SIMPLE)
+    assert mapping == {5: 0, 6: 1, 7: 2, 0: 3, 1: 4, 2: 5, 3: 6, 4: 7}
+
+    mapping = tested.NeuroM_section_to_NRN_section(SIMPLE, neurite_type="basal_dendrite")
+    assert mapping == {0: 0, 1: 1, 2: 2, 3: 3, 4: 4}
```

Comment:
The SIMPLE neuron doesn't seem to be sufficient to cover all the cases that affect the id mapping (valid type, zero length/no children, single child).

Reply:
yes... there was not tests before... I'll add more